Task1.ConsoleUI/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1.ConsoleUI
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string enteredString = "";
```

Comment:
My vision:

Console.Write("Enter your strings(To exit enter\"Exit\"): ");

string enteredString = Console.ReadLine();
for (; !IsExit(enteredString); enteredString = Console.ReadLine())
{
    Console.WriteLine(int.Parse(enteredString));
}

private static bool IsExit(string str)
{
    return String.IsNullOrWhiteSpace(str) || String.Compare(str, EXIT, StringComparison.InvariantCultureIgnoreCase) == 0;
}

From my point of view more readable.